system/lsmod.go

```diff
@@ -0,0 +1,32 @@
+package system
+
+import (
+	"github.com/yeticloud/yeti-discover/data"
+	"os/exec"
+	"regexp"
+	"runtime"
+	"strings"
+)
+
+func Lsmod(d *data.DiscoverJSON) { // make sure that the operating system is linux only
+	if runtime.GOOS != "linux" {
+		return
+	}
+	var cmd = exec.Command("bash", "-c", "lsmod | grep -v Module")
```

Comment:
@cwalsh003 why do you need... `bash -c` if lsmod is not in your path try `/usr/sbin/lsmod`